examples/oauth-pat-generation/netlify/functions/create-token.ts

```diff
@@ -0,0 +1,40 @@
+import type { Context, Config } from "@netlify/functions"
+
+export default async (req: Request, context: Context) => {
+  const accessToken = req.headers.get('Authorization')?.split(' ')[1]
+  
+  if (!accessToken) {
+    return new Response('Unauthorized', { status: 401 })
+  }
+
+  try {
+    // This endpoint Personal Access Tokens, or PATs, and act the same as the OAuth2 access tokens, but with a configurable expiration attached to them 
+    const response = await fetch('https://app.netlify.com/access-control/bb-api/api/v1/oauth/applications/create_token', {
```

Comment:
`https://app.netlify.com/access-control` is our proxy for our app. They should use `https://api.netlify.com/api/v1/oauth/applications/create_token`

Reply:
Thank you, updated